Automattic/wp-calypso
# Avoid broken flow and site re-creation when clicking back from checkout

Fixes https://github.com/Automattic/wp-calypso/issues/95024

## Proposed Changes

* Ensure that the same site is used when clicking back from checkout

## Why are these changes being made?


* Clicking back from checkout breaks the flow
* Clicking back again will create a new site

## Testing Instructions



* Go to Signup flow at `/setup/new-hosted-site`
* Add a plan to cart and reach checkout
* Click browser back button
* Select a plan again
* The same site slug should be used
* Complete the flow ( everything should work)
